src/editor.js

```diff
@@ -61,22 +67,27 @@ CKEDITOR.define( [
 		 *
 		 * The initialization consists of the following procedures:
 		 *
-		 *  * Load and initialize the configured plugins.
-		 *  * TODO: Add other procedures here.
+		 *  * Loads and initializes the configured plugins.
+		 *  * Fires the editor creator.
+		 *  * Initializes all editables.
 		 *
-		 * This method should be rarely used as `CKEDITOR.create` calls it one should never use the `Editor` constructor
-		 * directly.
+		 * This method should be rarely used as `CKEDITOR.create` calls it. One should never use the `Editor`
+		 * constructor directly.
 		 *
-		 * @returns {Promise} A promise which resolves once the initialization is completed.
+		 * @returns {Promise} A promise that resolves once the initialization is completed.
 		 */
 		init: function() {
 			var that = this;
 			var config = this.config;
 
 			// Create and cache a promise that resolves when all initialization procedures get resolved.
-			this._initPromise = this._initPromise || Promise.all( [
-				loadPlugins().then( initPlugins )
-			] );
+			this._initPromise = this._initPromise || Promise.resolve()
+				.then( loadPlugins )
+				.then( initPlugins )
+				.then( prepareData )
+				.then( fireCreator )
+				.then( initEditables )
```

Comment:
`initEditables` should be before `fireCreator`. Editables should be ready when we call creators, creator may need to ready some properties of editable.